src/main/resources/static/success.html

```
@@ -0,0 +1,87 @@
+                error: error.message
+            }, null, 2);
+        }
+    }
+
+    confirmPayment();
+</script>
+</body>
+</html>
+
+    confirmPayment();
+</script>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </script> ]

(tag-pair)